woocommerce-gutenberg-products-block.php

```diff
@@ -180,3 +180,27 @@ function woocommerce_blocks_get_i18n_data_json( $translations, $file, $handle, $
 }
 
 add_filter( 'pre_load_script_translations', 'woocommerce_blocks_get_i18n_data_json', 10, 4 );
+
+/**
+ * Filter translations so we can retrieve translations from Core when the original and the translated
+ * texts are the same (which happens when translations are missing).
+ *
+ * @param string $translation Translated text based on WC Blocks translations.
+ * @param string $text        Text to translate.
+ * @param string $domain      The text domain.
+ * @return string WC Blocks translation. In case it's the same as $text, Core translation.
+ */
+function woocommerce_blocks_get_php_translation_from_core( $translation, $text, $domain ) {
+	if ( 'woo-gutenberg-products-block' !== $domain ) {
+		return $translation;
+	}
+
+	// When translation is the same, that could mean the string is not translated.
+	// In that case, load it from core.
+	if ( $translation === $text ) {
+		return translate( $text, 'woocommerce' ); // phpcs:ignore
```

Comment:
What rule is being ignored here? Can we add specificity to the ignore rule?

Reply:
Actually it's three rules, I think it makes sense to ignore all of them in this context, but let me know if that doesn't sound good: 3c72d23c858d8dc00b8ea62ed7055df0fb035143.

Reply:
Ha! good catch Mike! You're probably wondering why I didn't say anything here :)

Albert, I generally prefer the rules to be on the line above what's being affected because it's easier to pick up when browsing the code.